pmemkv/README.md

```diff
@@ -16,46 +16,52 @@ LICENSE file.
 -->
 
 # PmemKV Driver for YCSB
-This driver is a binding for the YCSB facilities to operate against a PmemKV. It uses the PmemKV Java bindings.
+This driver is a binding for the YCSB facilities to operate against a [PmemKV](https://github.com/pmem/pmemkv). It uses the PmemKV Java bindings.
 
-## Quickstart
+## Quick Start
 
-### 1. Install PmemKV Java binding
-Optionally compile and install the [Java binding](https://github.com/pmem/pmemkv-java) with its dependencies:
-```shell
-export JAVA_HOME=#PATH_TO_YOUR_JAVA_HOME
-git clone https://github.com/pmem/pmemkv-java.git
-cd pmemkv-java
-mvn install
-```
+### 1. Install PmemKV Java Binding
+Optionally you can compile and install the latest version of PmemKV Java binding.
 
-### 2. Set up YCSB
+>Note: If you want to use custom installation, you'll have to update
+>this driver's `pom.xml`, changing `<artifactId>pmemkv-root</artifactId>`
+>into `<artifactId>pmemkv</artifactId>`!
+
+Simple follow [PmemKV Java installation instruction](https://github.com/pmem/pmemkv-java#installation),
+including at least:
+
+    export JAVA_HOME=#PATH_TO_YOUR_JAVA_HOME
+    git clone https://github.com/pmem/pmemkv-java.git
+    cd pmemkv-java
+    mvn install
+
+### 2. Set Up YCSB
 You need to clone the repository and compile PmemKV module.
 
-```
-git clone git://github.com/brianfrankcooper/YCSB.git
-cd YCSB
-mvn -pl site.ycsb:pmemkv-binding -am package
-```
+    git clone git://github.com/brianfrankcooper/YCSB.git
+    cd YCSB
+    mvn -pl site.ycsb:pmemkv-binding -am package
```

Comment:
Why not code block?
We can also use indentation in code sections.

Reply:
I've used the same approach as in other modules. For some reason, they use this style (which produces the same output as our ```sh approach - see for yourself: https://github.com/lukaszstolarczuk/YCSB/tree/use-pmemkv-from-maven/pmemkv)

Reply:
Okey, then LGTM